service/src/test/java/tds/session/web/endpoints/SessionControllerIntegrationTests.java

```diff
@@ -163,6 +165,28 @@ public void shouldReturnSessionAssessment() throws Exception {
     }
 
     @Test
+    public void shouldReturnSessionAssessments() throws Exception {
+        UUID sessionId = UUID.randomUUID();
+        SessionAssessment sessionAssessment1 = random(SessionAssessment.class);
+        SessionAssessment sessionAssessment2 = random(SessionAssessment.class);
+
+        UriComponents components = UriComponentsBuilder
+            .fromPath(String.format("/sessions/%s/assessment", sessionId)).build();
+
+        when(mockSessionService.findSessionAssessments(sessionId)).thenReturn(Arrays.asList(sessionAssessment1, sessionAssessment2));
+
+        http.perform(get(components.toUri())
+            .contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("[0].sessionId", is(sessionAssessment1.getSessionId().toString())))
+            .andExpect(jsonPath("[0].assessmentId", is(sessionAssessment1.getAssessmentId())))
+            .andExpect(jsonPath("[0].assessmentKey", is(sessionAssessment1.getAssessmentKey())))
+            .andExpect(jsonPath("[1].sessionId", is(sessionAssessment2.getSessionId().toString())))
+            .andExpect(jsonPath("[1].assessmentId", is(sessionAssessment2.getAssessmentId())))
+            .andExpect(jsonPath("[1].assessmentKey", is(sessionAssessment2.getAssessmentKey())));
```

Comment:
Can you @Autowire the ObjectMapper and make sure it converts properly?  I know it is a late addition that we've been slowly adding in our tests.

Not that I see a reason it will not just work.